fission/src/ui/modals/MainMenuModal.tsx

```diff
@@ -30,6 +33,29 @@ const MainMenuModal: React.FC<ModalPropsImpl & { startSingleplayerCallback: () =
                     }}
                     className="w-full my-1"
                 />
+                <Button
+                    value={"Load Default"}
+                    onClick={() => {
+                        closeModal()
+                        startSingleplayerCallback()
+                        MirabufCachingService.CacheRemote(
+                            "/api/mira/fields/FRC Field 2023_v7.mira",
+                            MiraType.FIELD
+                        ).then(cacheInfoField => {
+                            if (cacheInfoField) {
+                                SpawnCachedMira(cacheInfoField, MiraType.FIELD)
+                            }
+                        })
+                        MirabufCachingService.CacheRemote("/api/mira/robots/Dozer_v9.mira", MiraType.ROBOT).then(
+                            cacheInfoRobot => {
+                                if (cacheInfoRobot) {
+                                    SpawnCachedMira(cacheInfoRobot, MiraType.ROBOT)
+                                }
+                            }
+                        )
```

Comment:
The way these promises are set up means that the order isn't guaranteed and if the robot is spawned first, you have to configure/position it without seeing the field. I think it makes more sense to make sure the field spawns first. You can use `Promise.all(...)` here to do this and still let you download them both simultaneously